packages/karma-esm/demo/basic/karma.config.js

```diff
@@ -78,11 +78,9 @@ module.exports = config => {
       },
     },
 
-    coverageIstanbulReporter: {
-      reports: ['html', 'lcovonly', 'text-summary'],
+    coverageReporter: {
+      reporters: [{ type: 'html' }, { type: 'lcovonly' }, { type: 'text-summary' }],
```

Comment:
This might be the extend of it, but should we list a `migration.md` file in this PR so that users can easily make the version change here?

Reply:
`migration.md` file added. Hope I put it in the right place